Basic/Basic/MainViewController.m

```diff
@@ -20,13 +20,24 @@ @implementation MainViewController
 
 - (void) viewDidLoad
 {
-    self.organization = [[Organization alloc] initWithName:@"GGG company"];
-    
-    [self.organization addEmployeeWithName:@"Jim Bon"];
-    [self.organization addEmployeeWithName:@"Jack Li"];
-    [self.organization addEmployeeWithName:@"Nick Nu"];
-    [self.organization addEmployeeWithName:@"John Ve"];
     
+    NSEntityDescription *decs = [NSEntityDescription entityForName:@"OrganizationModel" inManagedObjectContext:self.managedObjectContext];
+    self.organization.name = @"bbn";
+    self.organization = [[Organization alloc] initWithEntity:decs insertIntoManagedObjectContext:self.managedObjectContext];
```

Comment:
Logical issue - you create an organization before even checking whether one exists.